core/src/main/java/org/jclouds/http/functions/ParseSax.java

```diff
@@ -93,7 +94,7 @@ private T convertStreamToStringAndParse(HttpResponse response) {
       String from = null;
       try {
          byte[] fromBytes = closeClientButKeepContentStream(response);
-         from = new String(fromBytes);
+         from = new String(fromBytes, StandardCharsets.UTF_8);
```

Comment:
I think this would be more correct to use the charset from the Content-Type header.  However I doubt any provider use charsets other than UTF-8 or ASCII?

Reply:
Probably not, but I can't say for sure.
I can get the charset from the Content-Type header (if it exists there) - are there some HTTP utilities already in the classpath I can use to avoid parsing the Content-Type string myself?